src/client/scripts/views/CreateAccount/CreateAccountContainer.js

```diff
@@ -8,6 +8,8 @@ import { HousingSituationView } from '../CreateAccount/HousingSituationView.js';
 import { NeedHousingView } from '../CreateAccount/NeedHousingView.js';
 import { HaveHousingView } from '../CreateAccount/HaveHousingView.js';
 import { Events } from '../../Events.js';
+import express from 'express';
```

Comment:
Express routing is used for HTTP and responding to client requests. This method will not change the URL. Also, Isha's working on routing, you can see her work in the PR's. If you google "change url without reloading page" many options will show up. Try following some of those.